graphistry/pygraphistry.py

```diff
@@ -670,6 +727,12 @@ def register(
                     import graphistry
                     graphistry.register(api=3, protocol='http', server='nginx', client_protocol_hostname='https://my.site.com', token='abc')
 
+        **Example: Databricks notebook and dashboard users need to call the following before register to correctly display the HTML link: 
```

Comment:
I think in the dashboard case, it needs to be in 2 different cells, one is a non-blocking url print, and the other the blocking register for whichever cell will plot

I'm not sure what's needed, if anything,  for notebooks?